cubic-server/protocol/ClientPackets.hpp

```diff
@@ -47,7 +48,13 @@ namespace protocol
     };
 
     std::shared_ptr<std::vector<uint8_t>> createPlayerChatMessage(const PlayerChatMessage &);
+    
+    struct DisconnectReason
+    {
+        std::string reason;
+    };
 
+    std::shared_ptr<std::vector<uint8_t>> DisconnectPlayerReason(const DisconnectReason &);
```

Comment:
I missed that on the first review, but this should be named like the rest of the functions here.

For example `createDisconnectPlayerReason`